README.md

```diff
@@ -4,127 +4,132 @@
 
 ## Introduction
 
-This helm chart installs the [Scalyr Agent monitor](https://app.scalyr.com/help/scalyr-agent) into a Kubernetes
-cluster. It supports installing the agent with all features required to support a Kubernetes cluster monitoring.
-Additionally, it can deploy Scalyr agents which monitor other parts of the infrastructure (for example a hosted
-database service etc.).
+This helm chart installs the [Scalyr Agent](https://app.scalyr.com/help/scalyr-agent) in a Kubernetes
+cluster. Two Agent Plugins plugins are also installed:
+- The [Kubernetes monitor](https://app.scalyr.com/monitors/kubernetes) enables the Agent to monitor Kubernetes clusters.
+- The [Kubernetes Events monitor](https://app.scalyr.com/monitors/kubernetes-events) collects events from the Kubernetes API server for all nodes except master.
 
-**Homepage:** <https://github.com/scalyr/helm-scalyr>
+We implement the Kubernetes-recommended node-level logging architecture. The Agent runs as a DaemonSet. An Agent pod runs on each node and collects logs from other pods on the node.
 
-## Installation
+By default, the Agent collects pod logs and container metrics for all nodes, and Kubernetes Events for all nodes except master. You can install the Agent to monitor other parts of the infrastructure, for example a hosted database service.
 
-Use
+
+## Installation 
+
+You must set some items:
+- ``scalyr.apiKey``: Must be a "Log Write Access" API key. Log into your DataSet account. Select your account (email address), then select "Api Keys".
+- ``scalyr.k8s.clusterName``: You must set a name for your Kubernetes cluster, which shows in the UI.
+- By default data uploads to our US server. For EU customers, set `scalyr.server="eu.scalyr.com"`.
+- `<release-version>`: The latest release version is at the top of the [README](https://scalyr.github.io/helm-scalyr/). You can consult the CHANGELOG for previous releases.  
+
+To install:
 
 ```bash
-helm install <name of release> scalyr-agent --repo https://scalyr.github.io/helm-scalyr/ --set scalyr.apiKey="<your write logs api key>" --set scalyr.k8s.clusterName="<your-k8s-cluster-name>"
```

Comment:
I'm somewhat confused about this "release version" change.

``helm install`` takes name of the application which will be installed aka this should be a unique application / installation / release name and not the release version.

If user wants to specify specific version, they need to use ``--version`` argument which is different from the "name" argument.

```bash

Usage:
  helm install [NAME] [CHART] [flags]
```

Reply:
Will revert. When I tested these commands `helm install v0.2.24 scalyr-agent ...` worked, and I was thinking that we should call this a "release version" instead of "name of release". After Googling "release" I see now that it is defined as "a version intended for use by the general population".

Reply:
Yeah pretty much anything will be accepted for the release name, but the release name is meant to represent unique name of that chart release / installation.

Technically user could have two scalyr-agent-2 charts installed on the same Kubernetes cluster (e.g. one in DaemonSet mode and one in Deployment mode running on a single node with some special configuration or similar).

In the case user could name 1 release / deployment / installation "scalyr-agent-2-daemonset" and other one "scalyr-agent-2-deployment" or similar.